### DISCUSSION TITLE
Are these servers legit or scam?

### DISCUSSION THREAD

**Discussion by magnemann** [dani-garcia/vaultwarden#3450] [Q&A]
First a big thanks to garcia and other working on this awesome projekt!

I've seen two public instances of vaultwarden.
~REDACTED~
**Can anyone confirm if they are legit and not a scammers front-end collecting pw?**
I guess there's no way of checking for scam other than a trust-chain...(?)
Do you know of more public instances ?

Thanks again, best regards, Magne.

**Comment by BlackDex** [comment]
We do not endorse any public instance.
Usage is at your own risk.
